apps/user-profile/src/assets/i18n/cs.json

```diff
@@ -350,6 +351,20 @@
           "CANCEL": "Zrušit",
           "SEND": "Poslat",
           "SUCCESS": "Chyba byla nahlášena a dostala lístek s číslem: "
+        },
+        "CHANGE_PASSWORD_DIALOG": {
+          "TITLE": "Změna hesla",
+          "CANCEL": "Zrušit",
+          "CHANGE": "Potvrdit",
+          "OLD_PASSWORD": "Staré heslo",
+          "NEW_PASSWORD": "Nové heslo",
+          "NEW_PASSWORD_AGAIN": "Znovu zadejte nové heslo",
+          "PWD_DONT_MATCH": "Hesla se nezhodují.",
+          "PWD_WEAK": "Heslo nesplňuje požadavky.",
+          "PWD_SHORT": "Heslo je příliž krátké.",
+          "FIELD_EMPTY": "Tohle políčko musí být vyplněné.",
+          "PASSWORD_INFO": "Heslo musí:\n - obsahovat pouze tiskové (bez přízvuku) znaky \n - být alespoň 10 znaků dlouhé\n - obsahovat nejméně 3 ze 4 skupin znaků:\n    - malá písmena\n    - velká písmena\n    - čísla\n    - speciální znaky",
```

Comment:
I'm not sure, what you mean by combination of words `only printing` and `non-accented` (in en.json). I think that it's two different things. `Only printing characters` I understand like no new lines/whitespaces (=_musí obsahovat pouze tisknutelné znaky_) and `non-accented` means no letters like á/ě/ú (=_písmena bez diakritiky_).